edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.Kubernetes.IntegrationTest/cluster/KindKubernetesClientProvider.cs

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Agent.Kubernetes.IntegrationTest.Cluster
+{
+    using System.Threading.Tasks;
+    using CliWrap;
+    using k8s;
+
+    public class KindKubernetesClientProvider : IKubernetesClientProvider
+    {
+        readonly string name;
+
+        public KindKubernetesClientProvider(string name)
+        {
+            this.name = name;
+        }
+
+        public async Task<IKubernetes> GetClient()
```

Comment:
`KindKubernetesClientProvider`  is providing the same function as `KindClusterManager`, and I didn't find this used do we need this?

Reply:
`KindClusterManager` allows to create/delete clusters with kind, whether `KindKubernetesClientProvider` provides a client for any cluster really.